sql/parser/eval.go

```diff
@@ -448,6 +448,8 @@ type EvalContext struct {
 	NodeID        uint32
 	StmtTimestamp time.Time
 	TxnTimestamp  time.Time
+	Location      string
```

Comment:
@vivekmenezes Not sure which comment you were responding to. I certainly wasn't suggesting making `sql/parser` depend on `sql`. If you added a `GetLocation func() *time.Location` function pointer to `EvalContext`, you could initialize that function pointer to parse the location from the session. `sql/parser` would effectively be calling into `sql` code, but there would be no dependency from `sql/parser` on the `sql` package.